Library/Sources/Manifest/ValidationStatusCode.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  ValidationStatusCode.swift
+//  C2PA
+//
+//  Created by Benjamin Erhart on 07.10.25.
+//
+
+import Foundation
+
+/**
+ https://spec.c2pa.org/specifications/specifications/1.0/specs/C2PA_Specification.html#_existing_manifests
```

Comment:
Let's refer to current spec:

```suggestion
https://spec.c2pa.org/specifications/specifications/2.2/specs/C2PA_Specification.html#_returning_validation_results
```